packages/host/app/services/render-service.ts

```diff
@@ -135,13 +135,22 @@ export default class RenderService extends Service {
           isNotLoadedError(e),
         ) as NotLoaded | undefined;
         if (isCardError(err) && err.status !== 500 && notLoaded) {
-          let linkURL = new URL(`${notLoaded.reference}.json`);
```

Comment:
this code was actually hiding a type issue--the unloaded references could be a string or an array of strings for links-to-many relationships. we want to handle both.